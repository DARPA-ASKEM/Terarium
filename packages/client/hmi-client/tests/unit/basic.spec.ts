import { assert, describe, expect, it } from 'vitest';

describe('basic tests to make sure it all works', () => {
	it('should have the correct square root', () => {
		assert.equal(Math.sqrt(4), 2);
	});

	it('should sum up to 2', () => {
		expect(1 + 1).eq(2);
	});

	it('should generate and match snapshot', () => {
		expect({ foo: 'bar' }).toMatchSnapshot();
	});
});
