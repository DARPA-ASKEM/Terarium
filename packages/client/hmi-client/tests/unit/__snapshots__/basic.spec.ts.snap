// Vitest Snapshot v1

exports[`basic tests to make sure it all works > should generate and match snapshot 1`] = `
{
  "foo": "bar",
}
`;
